pegasus/sites.v3/code.org/public/teacher-dashboard/review-hociyskvuwa/splat.haml

```diff
@@ -32,9 +32,11 @@ title: Review Hour of Code special events
 -results = []
 -events = []
 
--DB[:forms].where(kind:'HocSignup2015', review:nil).each do |i|
+-DB[:forms].where(kind:'HocSignup2015', review:nil).select(:name, :email, :data, :processed_data).distinct.each do |i|
```

Comment:
Does processed_data need to be unique? This is just the address of the location. What about events at the same school?